docs/API/configuration-guidelines.md

```diff
@@ -19,17 +19,26 @@ The default receiver buffer size is 8192 packets. It is approximately:
 
 ### Setting Receiver Buffer Size
 
-As already mentioned, the maximum allowed size of the receiver buffer is limited by the value of `SRTO_FC`.
 When the `SRTO_RCVBUF` option value is set using the `srt_setsockopt(..)` function,
-the provided size in bytes is internally converted to the corresponding size in packets
-using the configured value of the `SRTO_MSS` option to estimate the maximum possible payload of a packet.
+the provided size in bytes is internally converted to the corresponding size in packets.
+The size of a cell for a single packet in the buffer is defined by the
+`SRTO_MSS` option, which is 1500 by default.  This value, decreased by 28 in
+case of IPv4 (20 bytes for IPv4 header and 8 bytes for UDP header), gives 1472
+bytes per packet to be allocated. The actual memory occupied by the receiver
+buffer will be a multiplicity of that value. For the default 8192 packets it
+makes then 11776kB (11.5MB).
+
+Further, in every cell, there is 16 bytes of SRT header and the remaining space
+is for the payload.
+
+As already mentioned, the maximum allowed size of the receiver buffer is limited by the value of `SRTO_FC`.
 
 The following function returns the buffer size in packets:
```

Comment:
The following function returns the configured buffer size in packets depending on the `SRTO_RCVBUF`, `SRTO_MSS` and `SRTO_FC` values set:

Reply:
You meant:

```suggestion
The following function returns the configured buffer size in packets depending on the SRTO_RCVBUF, SRTO_MSS and SRTO_FC values set:
```